src/components/dialog/reply-dialog/ReplyDialog.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { HimanshuJPG } from "assets";
+import styles from "./ReplyDialog.module.css";
+import { Box, Typography } from "@mui/material";
+import { BROADCAST_MAX_CHARACTERS } from "utils";
+
+import {
+  AvatarGridBox,
+  BroadcastBoxHeader,
+  BroadcastDialogActions,
+  BroadcastDialogContainer,
+  BroadcastDialogContent,
+  DialogActionsCloseIcon,
+} from "components";
+
+// this variables are for design purpose only, they will get removed
+
+const post =
+  "Hey 👋 everyone\n\nfinally, I have completed all the basic features of the marvelsQuiz app\n\n🌐Live URL: https://marvelsquiz.vercel.app\n🔗GitHub link: https://github.com/hsnice16/react-marvelsQuiz\n\nfeedbacks are appreciated.";
```

Comment:
You could use template literals and lose those \n's.

Reply:
Actually, this is for styling purposes only. When I will integrate the backend, this data will come from API call.